Add reference to IS NULL operator from "Working with null" section (#1181)
neo4j/neo4j-documentation: cypher/cypher-docs/src/docs/dev/syntax/working-with-null.asciidoc

* Arithmetic expressions containing `null`: `1 + null`
* Function calls where any arguments are `null`: `sin(null)`

[[is-null-is-not-null]]
== Using `IS NULL` and `IS NOT NULL`

> Testing any value against `null` with both the `=` and the `<>` operators always evaluates to `null`.

Reviewer: This sounds to me like you were talking about applying both operators at once.
```suggestion
Testing any value against `null` with either the `=` or the `<>` operator always evaluates to `null`.
```

Reviewer 2: ```suggestion
Testing any value against `null`, either with the `=` operator or with the `<>` operator, always evaluates to `null`.
```

> Instead, the special `IS NULL`, or `IS NOT NULL`, equality operators should be used (see <<cypher-comparison>>).

Reviewer: ```suggestion
Instead, the special `IS NULL`, or `IS NOT NULL` equality operators should be used (see <<cypher-comparison>>).
```

Reviewer 2: ```suggestion
Therefore,  use the special equality operators `IS NULL` or `IS NOT NULL` instead (see <<cypher-comparison>>).
```
